test/extended/apibinding/apibinding.go

```diff
@@ -0,0 +1,46 @@
+package apibinding
+
+import (
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/kcp-dev/kcp-tests/test/extended/util"
+)
+
+var _ = g.Describe("[area-apiexports]", func() {
+	defer g.GinkgoRecover()
+
+	var (
+		k = exutil.NewCLIWithWorkSpace("kcp-apibinding")
+	)
+
+	g.It("Author:pewang-Critical-[Smoke][API] Verify APIBinding working with personal workspace", func() {
```

Comment:
Is there a discussion for adding label `[API]`?

Reply:
@Phaow you could add apibinding itself may be instead of API ?

Reply:
Hmm, I just use it origin title in polarion. From my understanding, maybe `[APIBinding]` is more suitable.@wangke19 As the case owner, WDYT?

Reply:
I'll remove the `[API]` label firstly. I'll add the suitable label back when we have a conclusion. Currently we could use the describe key words `api-exports` filter the cases of APIBinding.